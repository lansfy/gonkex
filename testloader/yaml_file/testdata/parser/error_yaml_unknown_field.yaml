- name: simple request test
  invalidfield: fail here
