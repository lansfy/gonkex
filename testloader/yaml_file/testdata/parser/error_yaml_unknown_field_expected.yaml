- Error: "unmarshal file testdata/parser/error_yaml_unknown_field.yaml: yaml: unmarshal errors:\n  line 2: field invalidfield not found in type yaml_file.TestDefinition"
